bin/runtests

```diff
@@ -1,471 +1,37 @@
-#! /usr/bin/env python
+#!/bin/sh
```

Comment:
To be sure, this should be `/bin/bash`.  On Ubuntu, `/bin/sh` is not `bash`.

Reply:
👍